seatunnel-connectors-v2/connector-file/connector-file-ftp/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/ftp/config/FtpConfigOptions.java

```diff
@@ -36,4 +36,9 @@ public class FtpConfigOptions extends BaseSourceConfigOptions {
             Options.key("host").stringType().noDefaultValue().withDescription("FTP server host");
     public static final Option<Integer> FTP_PORT =
             Options.key("port").intType().noDefaultValue().withDescription("FTP server port");
+    public static final Option<Boolean> FTP_ACTIVE_MODE_STATUS =
+            Options.key("active_mode_status")
+                    .booleanType()
+                    .defaultValue(true)
+                    .withDescription("FTP server mode default active");
```

Comment:
I think we should use enum to support 4 type mode.
![image](https://github.com/apache/seatunnel/assets/32387433/76ce4ca8-021a-47c7-8dc8-be0d0c6e1c7e)
Not only `true` or `false`.

Reply:
I am making the necessary modifications , thanks

Reply:
https://commons.apache.org/proper/commons-net/apidocs/org/apache/commons/net/ftp/FTPClient.html 
<img width="1715" alt="image" src="https://github.com/apache/seatunnel/assets/41814775/cac721c8-cb4d-4f83-8b85-c08943367656">

According to the above content, it can be seen that the Remote mode is server to server (FXP), while our product is in client and server mode, so currently only active is support  active_local 、passive_local , thank you very much

Reply:
I have created a new pull request and merged the commit. Thank you very much

Reply:
https://github.com/apache/seatunnel/pull/6099